src/Libraries/Microsoft.Extensions.AI.Abstractions/Utilities/AIJsonSchemaTransformOptions.cs

```diff
@@ -38,6 +38,11 @@ public sealed record class AIJsonSchemaTransformOptions
     /// </summary>
     public bool UseNullableKeyword { get; init; }
 
+    /// <summary>
+    /// Gets a value indicating whether to move the default keyword to the description field in the schema.
+    /// </summary>
+    public bool MoveDefaultKeywordToDescription { get; init; }
```

Comment:
This behavior was originally coupled to `RequireAllProperties` but I've now factored it into a standalone setting since they are logically independent.